pipeline_steps/common.groovy

```diff
@@ -2306,14 +2306,14 @@ void restartAbortedPRBuilds(String maint_date="today"){
  */
 void runThawIfSnapshot(){
   String thawFileName = "/gating/thaw/run"
-  def thawFile = new File(thawFileName)
-  if (thawFile.exists()){
-    stage('Execute snapshot thaw'){
-      sh """#!/bin/bash -xeu
+  stage('Execute snapshot thaw if file present'){
+    sh """#!/bin/bash -xeu
```

Comment:
This is a simple enough script that I think we can forgo setting -x here to cut down on some log noise.